helios-core/src/test/kotlin/helios/core/EnumTest.kt

```diff
@@ -0,0 +1,33 @@
+package helios.core
+
+import arrow.core.Left
+import arrow.core.Right
+import arrow.test.UnitSpec
+import helios.instances.EnumDecoderInstance
+import helios.instances.EnumEncoderInstance
+import io.kotlintest.shouldBe
+
+private enum class Foo {
+    A
+}
+
+internal class EnumTest : UnitSpec() {
+
+    init {
+        "helios serialization works both ways" {
```

Comment:
```suggestion
        "Enums should be encoded and decoded successfully" {
```

Reply:
I'll change this as requested, but I named it this way to follow the pattern of other tests.